mne/encoding/model.py

```diff
@@ -0,0 +1,329 @@
+import numpy as np
+from sklearn.linear_model import Ridge
+from sklearn.pipeline import Pipeline
```

Comment:
Yeah you can import from `sklearn` directly in examples. The reason for nesting in the package is taht it will prevent people from doing `import mne` if they don't have `sklearn`. But the examples and tutorials do not get imported so we don't need to nest there.